plugin/src/main/kotlin/tanvd/grazi/grammar/GrammarEngine.kt

```diff
@@ -50,9 +50,9 @@ object GrammarEngine {
     private fun getTyposSmall(str: String): LinkedSet<Typo> {
         if (isSmall(str)) return LinkedSet()
 
-        val lang = LangDetector.getLang(str, GraziConfig.get().enabledLanguages.toList()) ?: return LinkedSet()
+        val lang = LangDetector.getLang(str, GraziConfig.get().availableAndEnabledLanguages.toList()) ?: return LinkedSet()
```

Comment:
Why `toList`? Probably we need to get `Collection` or `Iterable` in getLang?

Reply:
Well, it getLang also used List